scripts/common.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Library to include in all wrapper runners.
+
+fail() {
+    (>&2 echo "$1")
+    exit 1
+}
+
+# Serve the index
+# ===============
+# It's probably more efficient to cat them together, but this way we see
+# if a given one is corrupted for any reason.
+
+serve_index() {
+    set -u
+    for e in $(ls ${INDEXER_OUTPUT_DIR}/*entries)
+    do
+        echo " * ${e}"
+        cat ${e} | /opt/kythe/tools/write_entries --graphstore ${INDEXER_OUTPUT_DIR}/gs
+    done
+    echo "== Converting to serving tables."
+    /opt/kythe/tools/write_tables \
+        --graphstore ${INDEXER_OUTPUT_DIR}/gs \
+        --out ${INDEXER_OUTPUT_DIR}/tbl \
+        --compress_shards
+    echo "== Starting HTTP server."
```

Comment:
could split to 2 funs: build_index for write_entries+write_tables, serve_index for the http serving